test/integration/testFirefoxWebext.js

```diff
@@ -91,4 +91,39 @@ describe('The Firefox Notes web extension', function() {
     expect(listNoteTitle).to.equal(paragraph);
   });
 
+  it('should be able to delete a note', async function() {
+    // Deletes a note
+    let listPage = await new ListPage(driver).waitForPageToLoad();
+    let notePage = await listPage.newNoteButton();
+    let title = "THIS IS A TEST";
+    await notePage.addNote(title)
+    listPage = await notePage.deleteNote();
+    let notesList = await listPage.notesList();
+    expect(notesList.length).to.equal(1)
+  });
+
+  it('should be able to add a note from the note page', async function() {
+    // Add a note from the Note Page
+    let listPage = await new ListPage(driver).waitForPageToLoad();
+    let notePage = await listPage.newNoteButton();
+    let title = "THIS IS A TEST";
+    let paragraph = "this isnt a test";
+    await notePage.addNote(title, paragraph)
+    expect(await notePage.noteTitle).to.equal(paragraph)
```

Comment:
A few more missing semicolons here, but maybe we're not running ESLint against the test directories.

This also had me a bit confused as it looks like we're asserting that the `noteTitle` is equal to the `paragraph` (I kind of expected it to equal the `title`, but I don't know the underlying data/objects).

Reply:
I think I need to change the nomenclature. The title and paragraph doesn't make much sense.

Reply:
@cedricium that helps a bunch actually. Thanks!

Reply:
Looks like I just had the 'enter' command in the wrong place.